### ISSUE TITLE
Holding up to double jump with tap jump enabled and inputting an aerial does not count as holding jump for purposes of disabling DJC

### ISSUE THREAD

**Issue by stingers135** [HDR-Development/HewDraw-Remix#2306]

**Comment by O1yyy** [comment]
Tap jump is exclusively a 1f Jump input to support frame perfect djc’s, it has its use and i advice not to change this behaviour

**Comment by stingers135** [comment]
Does tap jump behave like that in other games? Not being able to use it to cancel djc is pretty unintuitive

**Comment by O1yyy** [comment]
yep, exactly like other smash’s, people depend on it to execute perfect djc’s with less effort

**Comment by stingers135** [comment]
I'm unclear of what 1f input means in this instance. For one you can short or full hop with tap jump, is that not based off of some kind of frame count or is it just where your stick threshhold is when the controller is polled? 

Two, holding jump doesn't seem to continuously buffer jump inputs normally, yet it still functions to disable djc. Is that not what's meant by 1f input - that that input exists for 1f and is eliminated?

Unless you're saying tap jump was specifically coded to not disable djc to make getting djcs by pressing up easier? It's counterintuitive to how the other held jump inputs work, I don't see why tap jump should be on bumpers for one specific purpose. The behavior should match other jump inputs.
